pkg/api/componentreadiness/component_report.go

```diff
@@ -1347,20 +1706,46 @@ func (c *componentReportGenerator) generateComponentTestReport(baseStatus map[st
 		if !ok {
 			testStats.ReportStatus = crtype.MissingSample
 		} else {
-			var approvedRegression, baseRegression *regressionallowances.IntentionalRegression
+			requiredConfidence := c.getRequiredConfidence(testID.TestID, testID.Variants)
+			var approvedRegression *regressionallowances.IntentionalRegression
 			if len(c.VariantCrossCompare) == 0 { // only really makes sense when not cross-comparing variants:
 				// look for corresponding regressions we can account for in the analysis
 				approvedRegression = regressionallowances.IntentionalRegressionFor(c.SampleRelease.Release, testID.ColumnIdentification, testID.TestID)
-				baseRegression = regressionallowances.IntentionalRegressionFor(c.BaseRelease.Release, testID.ColumnIdentification, testID.TestID)
 				// ignore triage if we have an intentional regression
 				if approvedRegression == nil {
 					resolvedIssueCompensation, triagedIncidents = c.triagedIncidentsFor(testID)
 				}
 			}
-			requiredConfidence := c.getRequiredConfidence(testID.TestID, testID.Variants)
-			testStats = c.assessComponentStatus(requiredConfidence, sampleStats.TotalCount, sampleStats.SuccessCount,
-				sampleStats.FlakeCount, baseStats.TotalCount, baseStats.SuccessCount,
-				baseStats.FlakeCount, approvedRegression, baseRegression, resolvedIssueCompensation)
+
+			// this goes away and is just to validate
+			// that the fallback data for the base release
+			// isn't missing any tests
+			if c.cachedFallbackTestStatuses != nil {
+				// get fallback for the current base, make sure we don't have any misses'
+				if cachedTestStatuses, ok := c.cachedFallbackTestStatuses.Releases[c.BaseRelease.Release]; ok {
+					if cTestStats, ok := cachedTestStatuses.Tests[testIdentification]; ok {
+						// obviously we need more analysis here to determine if the
+						// previous release(s) stats should override
+						baseStats = cTestStats
+						baseReleaseMatches += 1
+					} else {
+						// log.Infof("Failed to find cache testId: %s - %v", baseStats.TestName, testIdentification)
+						baseReleaseMisses += 1
+					}
+				}
+
+			} else if !c.IgnoreFallback {
+				log.Error("Expected cached fallback statuses but was missing")
+			}
+
+			// this is where we look to see if a previous release has a higher pass rate
+			baseRelease := c.BaseRelease.Release
```

Comment:
Hmm, but it may not actually be the requestedBaseRelease, the next line down looks to return a fallback if that is determined higher threshold.  Would 'matchedBaseRelease' work / be clearer?